schema/spacewalk/common/tables/suseISSPeripheralChannels.sql

```diff
@@ -0,0 +1,24 @@
+--
+-- Copyright (c) 2024 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE TABLE suseISSPeripheralChannels
+(
+    id                          NUMERIC NOT NULL
+                                  CONSTRAINT suse_issperchan_id_pk PRIMARY KEY,
+    peripheral_id               NUMERIC NOT NULL
+                                  CONSTRAINT suse_issperchan_pid_fk
+                                    REFERENCES suseISSPeripheral(id) ON DELETE CASCADE,
+    peripheral_org_id           NUMERIC,
```

Comment:
We can use `INTEGER` (4B) or even `SMALLINT` (2B) here.

Reply:
I used INTEGER